src/diff/jsonDiff.ts

```diff
@@ -335,7 +334,6 @@ export function addNodeDiff(diff: object[]): NodeDiff {
  */
 export function addNodePropertyDiffs(
   key: string,
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Question, why were these calls not failing linting previously?

Reply:
Updated the file to avoid using a lot of `any`s. I replaced them with `unknown`s, which is not ideal, but better types will hopefully be added in the coming conversion to OO style.